crates/bevy_winit/src/lib.rs

```diff
@@ -158,6 +158,21 @@ fn change_window(
                         y: position[1],
                     });
                 }
+                bevy_window::WindowCommand::Center => {
+                    let window = winit_windows.get_window(id).unwrap();
+
+                    // What to do if current_monitor is None?
+                    // Abort?
+                    // Or use primary_monitor? And then what if that also is None?
+                    let screen_size = window.current_monitor().unwrap().size();
+
+                    let window_size = window.outer_size();
+
+                    window.set_outer_position(PhysicalPosition {
+                        x: (screen_size.width - window_size.width) as f64 / 2.,
+                        y: (screen_size.height - window_size.height) as f64 / 2.,
+                    });
+                }
```

Comment:
One thing I think I missed here is that current monitor might not be at (0,0). But I tested this on a second monitor at it works as expected, so now I'm confused.
(This also applies to centering on creation, but primary monitor will more often than not at (0,0))
If anyone has any idea what happens / what should happen, please tell

Reply:
I think it would be best to use primary_monitor, as I believe current_monitor changes. Maybe skip the problem by adding to the enum which monitor it should be centered in, so the user can choose?

Reply:
@Sheepyhead What do you mean?  
And I feel like just bypassing an issue wouldn't be great.

Reply:
> One thing I think I missed here is that current monitor might not be at (0,0). But I tested this on a second monitor at it works as expected, so now I'm confused.
As far as I know, the coordinates you specify are relative to the current monitor, so they always are at (0,0)

Reply:
@mockersf [winit docs](https://docs.rs/winit/0.26.1/winit/window/struct.Window.html#method.outer_position) say it's relative to "the top-left hand corner of the desktop" which "is not necessarily the same as the screen" (and this seems accurate).
I'm not sure how it works with changing monitors though.

Reply:
> @Sheepyhead What do you mean?
> And I feel like just bypassing an issue wouldn't be great.

Correct me if I'm wrong, but doesn't current_monitor grab the currently active monitor? And what I mean by skipping the problem is that the "centered" option lacks information, which is why this is an issue at all. Namely it lacks which monitor the window should be centered in. My suggestion was to add that to the enum data, so you let the user pick what monitor to center the window in, rather than assuming intent

Reply:
I like @Sheepyhead's idea here; we should probably let the user pick which monitor to center it in. Ideally use an Option of some sort, so that we can default to the primary monitor.